src/main/java/jmgomez/apipolicy/model/dto/PolicyDtoCov.java

```diff
@@ -14,5 +16,5 @@ public class PolicyDtoCov {
     @JsonProperty("descripcion")
     private String description;
     @JsonProperty("coberturas")
-    private String[] coverages;
+    private List<String> coverages;
```

Comment:
Check previous comments about our model. Clean unused properties, get rid of multiple DTOs if they don't serve a purpose, and don't use spanish in our API endpoints